wallet/src/Pos/Wallet/Web/Methods/Backup.hs

```diff
@@ -5,6 +5,8 @@
 module Pos.Wallet.Web.Methods.Backup
        ( importWalletJSON
        , exportWalletJSON
+       -- for testing purposes
+       , restoreWalletFromBackup
```

Comment:
This should not be needed  to export in order to test is `importWalletJSON` and `exportWalletJSON` work as they should